homeassistant/components/maxcube/__init__.py

```diff
@@ -112,3 +118,80 @@ def update(self):
                 self._updatets = time.monotonic()
             else:
                 _LOGGER.debug("Skipping update")
+
+    def disconnect(self):
+        """Disconnect from cube."""
+        with self.mutex:
+            self.cube.disconnect()
+
+
+async def _setup(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up using flow."""
+    config = entry.data
+
+    if DATA_KEY not in hass.data:
+        hass.data[DATA_KEY] = {}
+
+    # Get configs
+    host = config[CONF_HOST]
+    port = config[CONF_PORT]
+    scan_interval = config[CONF_SCAN_INTERVAL]
+
+    if host in hass.data[DATA_KEY]:
+        # Already configured, do nothing
+        _LOGGER.debug("Already configured, do nothing")
+        return True
+
+    try:
+        cube = MaxCube(host, port, now=now)
+        hass.data[DATA_KEY][host] = MaxCubeHandle(cube, scan_interval)
+    except timeout as ex:
+        _LOGGER.error("Unable to connect to Max!Cube gateway: %s", str(ex))
+        persistent_notification.create(
+            hass,
+            f"Error: {ex}<br />You will need to reload integration after fixing.",
+            title=NOTIFICATION_TITLE,
+            notification_id=NOTIFICATION_ID,
+        )
+        return False
+
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, Platform.CLIMATE)
```

Comment:
Please put this in a list called `PLATFORMS`

Reply:
8b31006d58c8579f4f6addec4b4b8f3a5ab28487